src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -760,7 +760,8 @@
             "StatusLightBrightness",
             "FeatureMap"
         ],
-        "Camera AV Settings User Level Management": ["MPTZPosition"]
+        "Camera AV Settings User Level Management": ["MPTZPosition"],
+        "Unit Localization": ["SupportedTemperatureUnits"]
```

Comment:
Doesn't need to be added here; list-typed attributes are automatically known to be AttributeAccessInterface.